tests/ApiRequestorTest.php

```diff
@@ -40,4 +40,29 @@ public function testHttpClientInjection()
         $injectedCurl = $method->invoke(new ApiRequestor());
         $this->assertSame($injectedCurl, $curl);
     }
+
+    public function testDefaultHeaders()
+    {
+        $reflector = new \ReflectionClass('Stripe\\ApiRequestor');
+        $method = $reflector->getMethod('_defaultHeaders');
+        $method->setAccessible(true);
```

Comment:
Oh man, TBH, I find that using reflection to access internal methods is testing practice that's kind of terrible on deep level, but I see that you're just following the existing convention here :$ Hopefully we can refactor this a bit later to be more palatable.